OOPBASICSPART1TASK6/src/com/gyk/java/oop/basics/Person.java

```diff
@@ -0,0 +1,62 @@
+package com.gyk.java.oop.basics;
+
+enum GENDER {MALE, FEMALE};

```

Comment:
It is valid to have enum inside a class but it is better to create separate class as an enum. But the name must be `Gender` instead of. `GENDER` and a semicolon is not required in a separate class

Reply:
ok, got it.